lib/core-generators/new/templates/config/custom.js.template

```diff
@@ -90,7 +90,16 @@ module.exports.custom = {
   *                                                                          *
   * Any other custom config this Sails app should use during development.    *
   *                                                                          *
-  ***************************************************************************/<% if (!caviar) { %>
+  ***************************************************************************/
+  
+  platformCopyrightYear: '2019',
+  platformName:  'NEW_APP_NAME',
+  platformCompanyName: 'NEW_APP_COMPANY_NAME',
+  platformCompanyAboutHref: 'https://sailsjs.com/about',
+  privacyPolicyUpdatedAt: 'July 16, 2019',
```

Comment:
Let's do all-caps things for these dates, too, so it's clearer that it needs to be updated ("PRIVACY_UPDATED_CALENDAR_DATE" or whatever makes sense)